packages/dashboards/tests/dummy/config/environment.js

```diff
@@ -31,7 +31,7 @@ module.exports = function(environment) {
       ],
       FEATURES: {
         enableDashboardExport: true,
-        multipleExportFileTypes: ['pdf', 'png'],
+        exportFileTypes: ['pdf', 'png'],
```

Comment:
It should be removed from dashboards as well right? @jkusa

Reply:
yes